plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/procedure/FlushJdbcMetadataCacheProcedure.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.jdbc.procedure;
+
+import com.google.common.collect.ImmutableList;
+import io.trino.plugin.jdbc.CachingJdbcClient;
+import io.trino.spi.procedure.Procedure;
+
+import javax.inject.Inject;
+import javax.inject.Provider;
+
+import java.lang.invoke.MethodHandle;
+
+import static io.trino.spi.block.MethodHandleUtil.methodHandle;
+import static java.util.Objects.requireNonNull;
+
+public class FlushJdbcMetadataCacheProcedure
+        implements Provider<Procedure>
+{
+    private static final MethodHandle FLUSH_JDBC_METADATA_CACHE = methodHandle(
+            FlushJdbcMetadataCacheProcedure.class,
+            "flushJdbcMetadataCache");
+
+    private final CachingJdbcClient cachingJdbcClient;
+
+    @Inject
+    public FlushJdbcMetadataCacheProcedure(CachingJdbcClient cachingJdbcClient)
+    {
+        this.cachingJdbcClient = requireNonNull(cachingJdbcClient, "cachingJdbcClient is null");
+    }
+
+    @Override
+    public Procedure get()
+    {
+        return new Procedure(
+                "system",
+                "flush_jdbc_metadata_cache",
```

Comment:
the procedure is registered in a JDBC connector, so `jdbc` is redundant in the name

also, "metadata" -- will we add pointed procedures for every single cache, if we have them?
do we want to discern `CachingJdbcClient`'s cache vs `CachingIdentifierMapping`'s cache?
I guess no.
But we still may want to keep "metadata" in the name, as all these is some kind of metadata.

cc @kokosing @hashhar @wendigo

Reply:
Yes I had doubts about it. I thought we could name both flush_metadata_cache. For JDBC and Hive.

Reply:
Renamed and moved to separate PR: https://github.com/trinodb/trino/pull/10265